### DISCUSSION TITLE
option to self-install Stereo Tool like SHOUTcast

### DISCUSSION THREAD

**Discussion by sb-200** [AzuraCast/AzuraCast#5278] [Ideas]
Hi,

Firstly, I would like to say thank you for providing such a fantastic piece of open-source software, and for the continuous improvements, it's a really great project.

I have a small suggestion to give, and that's to add the ability to self-install Stereo Tool, like how SHOUTcast can be installed. I'm not sure if this is even possible to implement but just wanted to leave this feedback as I feel it would be quite useful to a good few users :)

Thanks!

**Comment by BusterNeece** [comment]
@sb-200 Hello and thank you!

I believe we have an existing feature suggestion for adding StereoTool over at https://features.azuracast.com/; if so, I'd encourage you to upvote that suggestion to help inform us that more users want it to happen, and if not, feel free to create it!
